MGS2-MC/MGS2MemoryManager.cs

```diff
@@ -10,6 +11,7 @@ namespace MGS2_MC
 {
     internal static class MGS2MemoryManager
     {
+        private static int counter = 0;
```

Comment:
Was this counter just used for debugging purposes? I totally understand if so, but since this counter provides no actual function besides simply counting, it should not be committed. Same with the `System.Diagnostics` using

Reply:
That was a counter to make sure it checks for the MGS2 process once and only once.